content/en/data_streams/messages.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Messages
+---
+
+The Messages feature allows you to view messages at specific offsets within Kafka. Inspecting these messages can be crucial when troubleshooting issues related to specific services or data streams.
+
+## Setup
+Before diving into detailed configuration steps, here's a quick overview of what's involved in setting up the Messages feature:
+
+| Step               | Description                                                                                                                                       | Setup Guides              |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|
+| Agent setup        | An agent needs Kafka consumer integration enabled, be on at least version 7.69.0, and have remote configuration enabled                           | [Agent Setup][1]          |
+| Verify permissions | You'll need a few log access permissions which usually come standard as well as `Data Streams Monitoring Capture Messages` enabled in Datadog     | [Required Permissions][2] |
+
+
+### Agent Setup
+
+#### Selecting an Agent
+Any Datadog agent having access to Kafka can be used for setup.
+
+If you self-host Kafka, set up Messages on your Kafka broker's agent. Otherwise, choose any agent your producer or consumer services communicate with that has access to your Kafka cluster.
```

Comment:
set up `the kafka consumer check` on your Kafka broker's agent.

Also, there can be many brokers, I don't know what to recommend in this case 🤔 
Maybe we should say: `on your Kafka brokers agents` but then, it doesn't explain how to configure each one differently.